Source/IGListAdapter.m

```diff
@@ -641,19 +642,19 @@ - (NSIndexPath *)indexPathForSectionController:(IGListSectionController *)contro
     return attributes;
 }
 
-- (void)mapView:(UICollectionReusableView *)view toSectionController:(IGListSectionController *)sectionController {
+- (void)mapView:(__kindof UIView *)view toSectionController:(IGListSectionController *)sectionController {
```

Comment:
lets add a `IGListReusableView` protocol and conform table cells and collection cells?